node/src/accountant/payment_adjuster/adjustment_runners.rs

```diff
@@ -0,0 +1,254 @@
+        account
+            .analyzed_account
+            .qualified_as
+            .payment_threshold_intercept_minor = 3_000_000_000;
+        account.analyzed_account.qualified_as.creditor_thresholds =
+            CreditorThresholds::new(1_000_000_000);
+        account
+    }
+
+    #[test]
+    fn means_equal_requested_money_after_dsq_in_previous_iteration_to_return_capped_accounts() {
+        let subject = ServiceFeeOnlyAdjustmentRunner {};
+        let cw_service_fee_balance_minor = 10_000_000_000;
+        let mut payable_1 = weighted_payable_setup_for_surplus_test(111, 5_000_000_000);
+        payable_1.analyzed_account.disqualification_limit_minor = 3_444_333_444;
+        let mut payable_2 = weighted_payable_setup_for_surplus_test(222, 5_000_000_000);
+        payable_2.analyzed_account.disqualification_limit_minor = 3_555_333_555;
+        let expected_proposed_balance_1 = 3_444_333_444;
+        let expected_proposed_balance_2 = 3_555_333_555;
+
+        test_surplus_incurred_after_disqualification_in_previous_iteration(
+            subject,
+            payable_1,
+            payable_2,
+            cw_service_fee_balance_minor,
+            expected_proposed_balance_1,
+            expected_proposed_balance_2,
+        )
+    }
+
+    #[test]
+    fn means_become_bigger_than_requested_after_dsq_in_previous_iteration_to_return_capped_accounts(
+    ) {
+        let subject = ServiceFeeOnlyAdjustmentRunner {};
+        let cw_service_fee_balance_minor = 10_000_000_000;
+        let mut payable_1 = weighted_payable_setup_for_surplus_test(111, 5_000_000_000);
+        payable_1.analyzed_account.disqualification_limit_minor = 3_444_333_444;
+        let mut payable_2 = weighted_payable_setup_for_surplus_test(222, 4_999_999_999);
+        payable_2.analyzed_account.disqualification_limit_minor = 3_555_333_555;
+        let expected_proposed_balance_1 = 3_444_333_444;
+        let expected_proposed_balance_2 = 3_555_333_555;
+
+        test_surplus_incurred_after_disqualification_in_previous_iteration(
+            subject,
+            payable_1,
+            payable_2,
+            cw_service_fee_balance_minor,
+            expected_proposed_balance_1,
+            expected_proposed_balance_2,
+        )
+    }
+
+    #[test]
+    fn adjust_accounts_for_service_fee_only_runner_is_not_supposed_to_care_about_transaction_fee() {
+        let balance = 5_000_000_000;
+        let mut account = make_non_guaranteed_qualified_payable(111);
+        account.bare_account.balance_wei = balance;
+        let wallet_1 = make_wallet("abc");
+        let wallet_2 = make_wallet("def");
+        let mut account_1 = account.clone();
+        account_1.bare_account.wallet = wallet_1.clone();
+        let mut account_2 = account;
+        account_2.bare_account.wallet = wallet_2.clone();
+        let adjustment = Adjustment::TransactionFeeInPriority {
+            affordable_transaction_count: 1,
```

Comment:
Add a comment here to say that the limit is set here. Maybe you want to add more explanation.